docs/modules/ROOT/pages/how-tos/configuring/customizing-the-build.adoc

```diff
@@ -239,6 +336,90 @@ The task update PR will trigger the on-pull-request pipeline, testing the change
 
 By default, all pipelines push the images to a local repository that is set up as a part of installation. Ths registry address is registry-service.kind-registry:5001. It is not mandatory to use this local repo, so if you want to use your own Quay repo to control user permissions, you can do this by following link:https://github.com/konflux-ci/konflux-ci/blob/main/docs/quay.md#configuring-a-push-secret-for-the-build-pipeline[the instructions] for configuring a push secret for the build piepline.
 
+== Adding sidecar containers to the build pipeline
+
+You can add sidecar containers to the build pipeline to run additional tasks alongside the main container. For example, you can add a sidecar container to run a database to enable your unit tests to run against a real database.
+
+In the example, we add a sidecar container to task to run a PostgreSQL database. Such sidecar can be used to test database seeding, migrations, etc.
+
+[source,yaml]
+--
+spec:
+  tasks:
+  # ...
+   - name: test-db-seeding
+      params:
+        # such task is most likely using source code and will need a trusted artifact to not break the EC.
+        - name: SOURCE_ARTIFACT
+          value: $(tasks.clone-repository-oci-ta.results.SOURCE_ARTIFACT)
+        # database connection parameters, depend on the chosen image
+        - name: PGSQL_USER
+          value: "jdoe"
+        - name: PGSQL_PASSWORD
+          value: "foobar"
+        - name: PGSQL_HOSTNAME
+          value: "0.0.0.0"
+        - name: PGSQL_PORT
+          value: "5432"
+        - name: PGSQL_DATABASE
+          value: "db"
+      runAfter:
+        - clone-repository-oci-ta
+      taskSpec:
+        params:
+          - description: The Trusted Artifact URI pointing to the artifact with the application source code.
+            name: SOURCE_ARTIFACT
+            type: string
+        volumes:
+          - name: workdir
+            emptyDir: {}
+        stepTemplate:
+          volumeMounts:
+            - mountPath: /var/workdir
+              name: workdir
```

Comment:
This part is irrelevant to the sidecar that is added, right? It is just the "normal task definition"?

Reply:
Correct. I can just leave the sidecar config. And also link to an existing example if you would prefer it.